### PR TITLE
ru locale hacker directory

### PR DESCRIPTION
**Pull request by AlmazN**
Hacker directory added to lib/locales/ru with translation

per https://github.com/Marak/faker.js/issues/475 issue.

### CODE REVIEW

File: lib/locales/ru/hacker/abbreviation.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module["exports"] = [
```
Comment (Marak) [inline lib/locales/ru/hacker/abbreviation.js]: Is there a reason we need to include this file?

It seems we already have this information available in the fallback API ( `en` )?

One goal is not to duplicate data definitions in the project ( size ). Was fallback not working correctly if you omit this file in `ru` locale?

**Comment by Marak** [comment]
@AlmazN -

This looks very good, thank you!

Please see my note about abbreviations file. Can this be removed?

Looking forward to further contributions. :-)

File: lib/locales/ru/hacker/abbreviation.js (hunk 1 above)
Comment (AlmazN) [inline lib/locales/ru/hacker/abbreviation.js]: removing abbreviation in ru/locale leads to error

Comment (dhilt) [inline lib/locales/ru/hacker/abbreviation.js]: @AlmazN  you should just remove the link to this source at ./lib/locales/ru/hacker/index.js:3.
Remove require("./abbreviation") and default fallback will work...

**Comment by dhilt** [comment]
@AlmazN, congratulations! welcome to the open-source world community!

**Comment by Marak** [comment]
Thank you both for your contributions!

It's really good to see locale data improving with help of native speakers.

Cheers!

**Pull request merged**